frontend/packages/kubevirt-plugin/src/selectors/vm/vmlike.ts

```diff
@@ -0,0 +1,22 @@
+import { K8sKind, TemplateKind } from '@console/internal/module/k8s';
+import { TemplateModel } from '@console/internal/models';
+import { VMLikeEntityKind, VMKind } from '../../types';
+import { VirtualMachineModel } from '../../models';
+import { selectVM } from '../vm-template/selectors';
+
+export const isVM = (vmLikeEntity: VMLikeEntityKind): boolean =>
```

Comment:
this could be a type guard
```suggestion
export const isVM = (vmLikeEntity: VMLikeEntityKind): vmLikeEntity is VMKind =>
```

Reply:
good idea, done